egs/thchs30/s5/local/score.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# Copyright 2012  Johns Hopkins University (Author: Daniel Povey)
+# Apache 2.0
+
```

Comment:
This while script should perhaps be replaced by a link to steps/score_kaldi.sh
which does the same thing